pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[tool.poetry]
+name = "lusidtools"
+version = "2.0.0a0"
```

Comment:
does this need to go to just "2.0.0"?

Reply:
This is the prerelease tag - the pipeline will patch it to 2.0.0 automatically before deployment.

This way the version in source control matches the deployed version